Gemfile

```diff
@@ -1,15 +1,15 @@
 source 'https://rubygems.org'
-gem 'fedora-migrate', git: 'https://github.com/projecthydra-labs/fedora-migrate.git'
+gem 'fedora-migrate', '~> 0.5.0'
 gem 'rdf-rdfxml'
 
 gem 'hydra-head', '~> 10.3.4'
 gem 'active-fedora', '>= 10.3.0'
 gem 'active_fedora-datastreams'
 gem 'active_fedora-noid', '~> 2.0.0'
-gem 'speedy-af', '~> 0.1.0', git: 'https://github.com/projecthydra-labs/speedy_af.git'
+gem 'speedy-af', git: 'https://github.com/projecthydra-labs/speedy_af.git', tag: 'v0.1.0'
 gem 'blacklight', '~> 6.6'
 gem 'rdf', '~> 2.1.1'
-gem 'ldp', git:'https://github.com/projecthydra/ldp.git'
+gem 'ldp', git:'https://github.com/projecthydra/ldp.git', tag: 'v0.6.4'
```

Comment:
Looks like we pegged this to master to get a bug patch, but using the 0.6.4 released gem should be fine or even taking this line out entirely might work.